DocGen4/Output/Module.lean

```diff
@@ -86,18 +86,19 @@ def docInfoHeader (doc : DocInfo) : HtmlM Html := do
 /--
 The main entry point for rendering a single declaration inside a given module.
 -/
-def docInfoToHtml (module : Name) (doc : DocInfo) : HtmlM Html := do
```

Comment:
Similar pattern here, all of this reference pull through should be a state monad.